taxonomies/food/ingredients.txt

```diff
@@ -4799,6 +4801,7 @@ hr:blagi punomasni jogurt
 
 <en:yogurt
 de:Joghurterzeugnis
+nova:en:4
```

Comment:
"Yogurt product" seems to be yogurt plus things like starch and ferments, it should not be NOVA 4.
e.g. "Joghurterzeugnis (9%) (fettarme Milch, Stärke, Joghurtkulturen)"

Reply:
In fact we do not know what happend to it. It is not simple yoghurt. If you look only at the ingredients you might miss hidden processing. What do we do with that? What is said about that?

Reply:
removed for now